subtitles/observer.lua

```diff
@@ -194,9 +198,23 @@ self.get_timing = function(position)
     return -1
 end
 
-self.collect = function()
+self.collect = function(n_lines)
```

Comment:
This function does two different things depending on the argument passed. In such cases it's better to have two functions instead of one.

Reply:
Thanks, good point.